pom.xml

```diff
@@ -1,99 +1,114 @@
-            <artifactId>nexus-repository</artifactId>
-            <scope>provided</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-compress</artifactId>
-        </dependency>
-
-        <dependency>
-            <groupId>org.sonatype.goodies</groupId>
-            <artifactId>goodies-testsupport</artifactId>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.skyscreamer</groupId>
-            <artifactId>jsonassert</artifactId>
-            <version>1.5.0</version>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.sonatype.nexus</groupId>
-            <artifactId>nexus-rapture</artifactId>
-            <scope>provided</scope>
-        </dependency>
-        
-    </dependencies>
-
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.sonatype.nexus.buildsupport</groupId>
-                <artifactId>extjs-maven-plugin</artifactId>
-                <configuration>
-                    <namespace>NX.composer</namespace>
-                </configuration>
-            </plugin>
-
-            <plugin>
-                <groupId>org.sonatype.plugins</groupId>
-                <artifactId>yuicompressor-maven-plugin</artifactId>
-            </plugin>
-
-            <plugin>
-                <groupId>org.apache.karaf.tooling</groupId>
-                <artifactId>karaf-maven-plugin</artifactId>
-            </plugin>
-
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.20</version>
-            </plugin>
-
-        </plugins>
-    </build>
+    <modules>
+        <module>nexus-repository-composer</module>
+        <module>nexus-repository-composer-it</module>
+    </modules>
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+    </distributionManagement>
+    <properties>
+        <nxrm-version>3.19.0-SNAPSHOT</nxrm-version>
+    </properties>
+    <repositories>
+        <!-- ensure we can find the parent pom when starting from an empty local .m2 repository -->
+        <repository>
+            <id>rso-snapshots</id>
+            <url>https://repository.sonatype.org/content/repositories/snapshots</url>
+        </repository>
+    </repositories>
+    <profiles>
+        <profile>
+            <id>deploy</id>
+            <!--
+            only sign during deploy phase
+            -->
+            <activation>
+                <property>
+                    <name>performRelease</name>
+                    <value>true</value>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <!--
```

Comment:
The `maven-enforcer-plugin` needs to be moved out of the \<profile> and into the "main" \<build> section, a la: https://github.com/sonatype-nexus-community/nexus-format-archetype/blob/master/src/test/resources/projects/it1/reference/pom.xml#L58

Reply:
in: 5f4baf6